### PR TITLE
spanish translation for advanced/index.md

### PR DESCRIPTION
**Pull request by jfunez**

### CODE REVIEW

File: docs/es/docs/advanced/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Guía de usuario avanzada - Introducción
+
+## Funcionalidades Adicionales
+
+El [Tutorial - Guía de Usuario](../tutorial/){.internal-link target=_blank} principal debe ser suficiente para darte un paseo por todas las funcionalidades principales de **FastAPI**
+
+En las secciones siguientes verás otras opciones, configuraciones, y funcionalidades adicionales.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/index.md:7]:
```suggestion
En las secciones siguientes verás otras opciones, configuraciones, y características adicionales.
```

File: docs/es/docs/advanced/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Guía de usuario avanzada - Introducción
+
+## Funcionalidades Adicionales
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/index.md:3]:
```suggestion
## Características Adicionales
```

File: docs/es/docs/advanced/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+El [Tutorial - Guía de Usuario](../tutorial/){.internal-link target=_blank} principal debe ser suficiente para darte un paseo por todas las funcionalidades principales de **FastAPI**
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/index.md:5]:
```suggestion
El [Tutorial - Guía de Usuario](../tutorial/){.internal-link target=_blank} principal debe ser suficiente para darte un paseo por todas las características principales de **FastAPI**
```

File: docs/es/docs/advanced/index.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+En las secciones siguientes verás otras opciones, configuraciones, y funcionalidades adicionales.
+
+!!! tip
+    Las próximas secciones **no son necesariamente "avanzadas"**.
+
+    Y es posible que para tu caso, la solución se encuentre en una de estas.
+
+## Lea primero el Tutorial
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/index.md:14]:
```suggestion
## Lee primero el Tutorial
```

File: docs/es/docs/advanced/index.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Tu puedes continuar usando la mayoría de las funcionalidades de **FastAPI** con el conocimiento del [Tutorial - Guía de Usuario](../tutorial/){.internal-link target=_blank} principal.
+
+Y en las siguientes secciones se asume que tu has leído y se asume que tu conoces las ideas principales.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/index.md:18]:
```suggestion
Puedes continuar usando la mayoría de las características de **FastAPI** con el conocimiento del [Tutorial - Guía de Usuario](../tutorial/){.internal-link target=_blank} principal.

En las siguientes secciones se asume que lo has leído y conoces esas ideas principales.
```

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thank you for your work @jfunez ! 🚀 👍 

I suggested some minor changes. Mainly the change from "funcionalidades" to "características" so we can maintain consistency with the earlier translations.

**Comment by jfunez** [comment]
thanks for the review @mariacamilagl 
I will adjust and review the other PRs I sent the apply your suggestions there also.

File: docs/es/docs/advanced/index.md (hunk 4 above)
Comment (AdrianDeAnda) [inline docs/es/docs/advanced/index.md:14]: Maybe an infinitive verb would be more suited?

```suggestion
## Leer primero el Tutorial
```

**Review by mariacamilagl — APPROVED** [review]
Thank you for implementing the changes @jfunez ! 🚀

**Comment by tiangolo** [comment]
Awesome! Thank you @jfunez ! :rocket: :tada: 

And thanks @AdrianDeAnda and @mariacamilagl for your reviews! :nerd_face: :memo:

**Pull request merged**